linker_scripts/object_script.txt

```diff
@@ -84,3 +84,13 @@ D_0600AB90 = 0x0600AB90;
 D_06007750 = 0x06007750;
 D_0600A390 = 0x0600A390;
 D_0600A490 = 0x0600A490;
+
+/* z_en_scopecoin */
+D_04061FC0 = 0x04061FC0;
+D_04061FE0 = 0x04061FE0;
+D_04062000 = 0x04062000;
+D_04062040 = 0x04062040;
+D_04062020 = 0x04062020;
+D_04062060 = 0x04062060;
+D_04062000 = 0x04062000;
+D_040622C0 = 0x040622C0;
```

Comment:
These addresses are from segment 4, `gameplay_keep`, so they should go with the other `0x04` addresses up top.